typescript/agentkit/package.json

```diff
@@ -43,7 +59,8 @@
     "@alloralabs/allora-sdk": "^0.1.0",
     "@coinbase/coinbase-sdk": "^0.20.0",
     "@jup-ag/api": "^6.0.39",
-    "@privy-io/server-auth": "^1.18.4",
+    "@privy-io/public-api": "2.20.5",
```

Comment:
`@privy-io/server-auth` indirectly replied on `@privy-io/public-api`, but did not directly add the dependency to its package. This resulted in fallback behavior, and in cjs, the incorrect package was being grabbed.

Solved by explicitly adding the dependency ourself, and selecting the version that `server-auth@1.19.3` depends on based on the privy repo's lock file.